src/encoded/schemas/ivp_a1.json

```diff
@@ -0,0 +1,682 @@
+{
+  "title": "NACC Uniform Data Set (UDS) - INITIAL VISIT PACKETFORM A1: SUBJECT DEMOGRAPHICS ",
+  "description": "INSTRUCTIONS: This form is to be completed by intake interviewer based on ADC scheduling records, subject interview, medical records, and proxy co-participant report (as needed). For additional clarification and examples, see UDS Coding Guidebook for Initial Visit Packet, Form A1. Check only one box per question.",
+  "comment": "Schema for submitting UDS_ivp_A1 form information.",
+  "id": "/profiles/ivp_a1.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "identifyingProperties": [
+    "uuid"
+  ],
+  "required": [
+    "reason",
+    "refersc",
+    "prestat",
+    "prespart",
+    "source",
+    "birthmo",
+    "birthyr",
+    "sex",
+    "hispanic",
+    "race",
+    "racesec",
+    "raceter",
+    "primlang",
+    "educ",
+    "maristat",
+    "independ",
+    "residenc",
+    "zip",
+    "handed"
+  ],
+  "additionalProperties": false,
+  "mixinProperties": [
+    {
+      "$ref": "mixins.json#/schema_version"
+    },
+    {
+      "$ref": "mixins.json#/submitted"
+    },
+    {
+      "$ref": "mixins.json#/standard_status"
+    },
+    {
+      "$ref": "mixins.json#/uuid"
+    }
+  ],
+  "type": "object",
+  "properties": {
+    "schema_version": {
+      "default": "1"
+    },
+    "reason": {
+      "title": "1. Primary reason for coming to ADC:",
+      "type": "string",
+      "enum": [
+        "1 To participate in a research study",
+        "2 To have a clinical evaluation",
+        "3 Other",
+        "4 Both (to participate in a research study and to have a clinical evaluation)",
+        "9 Unknown"
+      ]
+    },
+    "refersc": {
+      "title": "2a. Principal referral source:",
+      "description": "(if answer is 1 or 2, CONTINUE TO QUESTION 2B; otherwise, SKIP TO QUESTION 3.)",
+      "type": "string",
+      "enum": [
+        "1 Self-referral",
+        "2 Non-professional contact (spouse/partner, relative, friend, coworker, etc.)",
+        "3 ADC participant referral",
+        "4 ADC clinician, staff, or investigator referral",
+        "5 Nurse, doctor, or other health care provider",
+        "6 Other research study clinician/staff/investigator (non-ADC; e.g., ADNI, Women's Health Initiative)",
+        "7 Non-ADC media appeal (e.g., Alzheimer's Association)",
```

Comment:
2a, 14,17 all have version related field, need to add dependency later after merged with header.